### PR TITLE
Slack notification to include unittests

### PR DESCRIPTION
**Pull request by Shellyber**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/18846

## Does it break backward compatibility?
   - No

## Must have
- [ ] Code Review

### CODE REVIEW

File: Tests/scripts/run_parallel_pkg_dev_tasks.py [hunk 1]
```diff
@@ -4,8 +4,11 @@
 import subprocess
 import concurrent.futures
 from typing import List, Optional, Tuple
+
 from pkg_dev_test_tasks import get_dev_requirements
 
+DEMISTO_GREY_ICON = 'https://3xqz5p387rui1hjtdv1up7lw-wpengine.netdna-ssl.com/wp-content/uploads/2018/07/Demisto-Icon-Dark.png'
```
Comment (Itay4) [inline Tests/scripts/run_parallel_pkg_dev_tasks.py]: why is this needed here?

File: Tests/scripts/slack_notifier.py [hunk 2]
```diff
@@ -40,9 +40,9 @@ def options_handler():
 
 
 def get_attachments(build_url, env_results_file_name):
-    content_team_fields, content_fields, failed_tests = get_fields()
+    content_team_fields, content_fields, failed_tests, failed_unittests = get_fields()
     color = 'good' if not failed_tests else 'danger'
-    title = 'Content Build - Success' if not failed_tests else 'Content Build - Failure'
+    title = 'Content Build - Success' if not (failed_tests or failed_unittests) else 'Content Build - Failure'
```
Comment (Itay4) [inline Tests/scripts/slack_notifier.py]: shouldn't the operator in the condition be `and` ?

File: Tests/scripts/run_parallel_pkg_dev_tasks.py [hunk 3]
```diff
@@ -82,6 +90,7 @@ def main():
             res = future.result()
             handle_run_res(res, fail_pkgs, good_pkgs)
     if fail_pkgs:
+        create_result_files(fail_pkgs)
```
Comment (Itay4) [inline Tests/scripts/run_parallel_pkg_dev_tasks.py]: can you explain the flow here?

File: Tests/scripts/run_parallel_pkg_dev_tasks.py [hunk 4]
```diff
@@ -4,8 +4,11 @@
 import subprocess
 import concurrent.futures
 from typing import List, Optional, Tuple
+
```
Comment (Itay4) [inline Tests/scripts/run_parallel_pkg_dev_tasks.py]: remove empty line

File: Tests/scripts/run_parallel_pkg_dev_tasks.py [hunk 5]
```diff
@@ -4,8 +4,11 @@
...
 from pkg_dev_test_tasks import get_dev_requirements
 
+DEMISTO_GREY_ICON = 'https://3xqz5p387rui1hjtdv1up7lw-wpengine.netdna-ssl.com/wp-content/uploads/2018/07/Demisto-Icon-Dark.png'
+
```
Comment (Itay4) [inline Tests/scripts/run_parallel_pkg_dev_tasks.py]: remove empty line

File: Tests/scripts/run_parallel_pkg_dev_tasks.py (hunk 1 above)
Comment (Shellyber) [reply Tests/scripts/run_parallel_pkg_dev_tasks.py]: It's not, removing it now

File: Tests/scripts/run_parallel_pkg_dev_tasks.py (hunk 3 above)
Comment (Shellyber) [reply Tests/scripts/run_parallel_pkg_dev_tasks.py]: I want to get only the fail packages. In the circle it will look like this:
image

So I'm taking this packages and creating a file that later the slack_notifier will read and execute.

File: Tests/scripts/slack_notifier.py (hunk 2 above)
Comment (Shellyber) [reply Tests/scripts/slack_notifier.py]: No, because if one of the failures won't exist and the other yes, it will return a success with the `and` operator.

**Comment by Shellyber** [comment]
Also will look like this:
image

File: Tests/scripts/run_parallel_pkg_dev_tasks.py (hunk 3 above)
Comment (Itay4) [reply Tests/scripts/run_parallel_pkg_dev_tasks.py]: ok got it
lets improve function name,and add docstring to it

**Comment by Itay4** [comment]
@Shellyber looks good 👍 
see my comment above
